target/linux/ar71xx/files/arch/mips/ath79/mach-ap531b0.c

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Drop this line, please.
Also, make sure this file doesn't have executable attribute set.